app/styles/main.scss

```diff
@@ -642,31 +642,31 @@ h5 {
 
 .input-group-addon.primary {
   color: rgb(255, 255, 255);
-  background-color: rgb(50, 118, 177);
+  background-color: $brand-primary;
   border-color: rgb(40, 94, 142);
 }
 
 .input-group-addon.success {
   color: rgb(255, 255, 255);
-  background-color: rgb(92, 184, 92);
+  background-color: $brand-success;
   border-color: rgb(76, 174, 76);
 }
 
 .input-group-addon.info {
```

Comment:
Just curious, when a tag with `info` `primary` etc class name, isn't it automatically getting the color from bootstrap？

Reply:
For boostrap style, I don't want everything but color, so I just use $color directly.

Reply:
These styles all have been predefined by bootstrap. I'm not sure why you want to overwrite them?

Reply:
I tried `bg-danger` but its color is lighter than we want so I just overwrite it. I didn't find out a css called `danger`.